src/test/data/quarkusio/_data/versioned/latest/index/quarkus.yaml

```diff
@@ -0,0 +1,2828 @@
+# Generated file. Do not edit
```

Comment:
Did you include this on purpose? It's supposed to be in the sample ZIP already, no?

Reply:
yes, it's in the zip. but since there're a few of those ascidoc files present in that directory I was thinking that there should also be a copy of the yaml (for reasons unknown 😄)

Reply:
> since there're a few of those ascidoc files present in that directory

That's... not normal. I think I forgot to remove these.

I'll create a separate PR with your changes and a few more.

Reply:
ohh ok thanks 😃 
and sorry I've pushed a change removing that type-mapping and added `Awaitility.await().timeout(Duration.ofMinutes(5))` to see if that will help with the tests on GH since they work ok locally ...

Reply:
> I'll create a separate PR with your changes and a few more.

Done: https://github.com/quarkusio/search.quarkus.io/pull/55